### PR TITLE
Training set should be test set in boostrapping lesson

### LINKED ISSUE #26: Training set should be test set in boostrapping lesson
In the following section we evaluate the performance on the training set.

https://github.com/carpentries-incubator/machine-learning-novice-python/blob/8b353791dfd982f8c2eb79174ce6e575fb6d2512/_episodes/07-bootstrapping.md?plain=1#L71-L77

 I think we should use the test set here.

### PR DESCRIPTION
**Pull request by ttohya**
As discussed in #26, the bootstrapping lesson evaluates the performance on the training set instead of the test set.

This pull request :

- changes training set to test set 
- updates figures to use test set

Closes #26

### CODE REVIEW

**Comment by tompollard** [comment]
Looks good to me, thanks Takeshi!
